src/technologies/n.json

```
@@ -667,6 +667,21 @@
     },
     "website": "https://nextjs.org"
   },
+  "Next.js App Router": {
+    "cats": [
+      12
+    ],
+    "description": "The Next.js App Router is a new paradigm for building applications using React's latest features.",
+    "icon": "Next.js.svg",
+    "implies": [
+      "Next.js"
+    ],
+    "js": {
+      "next.AppDir": "true"
```

```suggestion
      "next.appDir": "true"
```

It's a lowercase `a`